src/tracking/trackers/progressive-profiling.js

```diff
@@ -0,0 +1,37 @@
+import { createEventTracker, createPageEventTracker } from '../../data/segment/utils';
+
+export const eventNames = {
+  progressiveProfilingSubmitClick: 'edx.bi.welcome.page.submit.clicked',
+  progressiveProfilingSkipLinkClick: 'edx.bi.welcome.page.skip.link.clicked',
+  disablePostRegistrationRecommendations: 'edx.bi.user.recommendations.not.enabled',
+  progressiveProfilingSupportLinkCLick: 'edx.bi.welcome.page.support.link.clicked',
+  loginAndRegistration: 'login_and_registration',
+};
+
+// Event link tracker for Progressive profiling skip button click
+export const trackProgressiveProfilingSkipLinkClick = evenProperties => createEventTracker(
+  eventNames.progressiveProfilingSkipLinkClick, { ...evenProperties },
+)();
+
+// Event tracker for progressive profiling submit button click
+export const trackProgressiveProfilingSubmitClick = (evenProperties) => createEventTracker(
+  eventNames.progressiveProfilingSubmitClick,
+  { ...evenProperties },
+)();
+
+// Event tracker for progressive profiling submit button click
+export const trackDisablePostRegistrationRecommendations = (evenProperties) => createEventTracker(
+  eventNames.disablePostRegistrationRecommendations,
+  { ...evenProperties },
+)();
+
+// Tracks the progressive profiling page event.
+export const trackProgressiveProfilingPageViewed = () => {
+  createPageEventTracker(eventNames.loginAndRegistration, 'welcome')();
+};
+
+// Tracks the progressive profiling spport link click.
+export const trackProgressiveProfilingSupportLinkCLick = () => createEventTracker(
```

Comment:
discussed this with @zainab-amir, since we are using [RedirectLogistration](https://github.com/openedx/frontend-app-authn/blob/c1ec9b6e994e19948c8886b297c7202638420fd4/src/progressive-profiling/ProgressiveProfiling.jsx#L206), we shouldn't use `createLinkTracker` here as it might interrupt our current functionality

Reply:
we'll do this separately